### PR TITLE
User's Profile edit completed

### PR DESCRIPTION
**Pull request by Shwetakale**
Document upload completed. Sending notification mail when user updates profile photo, Designation, Joining date completed

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -12,3 +12,5 @@ Gemfile.lock
 README.rdoc
 spec/dummy/README.rdoc
 spec/dummy/public/uploads/
+app/assets/images/
```
Comment (kirandc) [inline .gitignore]: Why app assets images in .gitignor

File: Gemfile [hunk 2]
```diff
@@ -35,6 +35,7 @@ group :test do
   gem 'mongoid-rspec'
   gem 'factory_girl_rails'
   gem 'database_cleaner'
+  gem 'simplecov', '~> 0.7.1'
```
Comment (kirandc) [inline Gemfile:38]: simplecove should be in test group

File: .gitignore (hunk 1 above)
Comment (Shwetakale) [inline .gitignore]: Sir, for that default fallback images.. Should i remove that?

File: Gemfile (hunk 2 above)
Comment (Shwetakale) [inline Gemfile:38]: sir simplecove is in test group only.

File: app/models/office_automation_employee/user.rb [hunk 3]
```diff
@@ -93,5 +97,14 @@ def invite_by_csv(file)
       end
       [invalid_rows, $. - 1]  # $. is last row number from file
     end
+
+    def send_mail
+      @updated_attributes = {}
```
Comment (kirandc) [inline app/models/office_automation_employee/user.rb]: Wrong implementation Set image, joining_date, designation in Constant and check if any attributes change then send mail. In future if I need add one more field so then I don't need to add code again.

**Pull request merged**